src/debug/di/breakpoint.cpp

```diff
@@ -57,9 +57,11 @@ HRESULT CordbBreakpoint::BaseIsActive(BOOL *pbActive)
  * ------------------------------------------------------------------------- */
 
 CordbFunctionBreakpoint::CordbFunctionBreakpoint(CordbCode *code,
-                                                 SIZE_T offset)
+                                                 SIZE_T offset,
+                                                 BOOL bindToAllNativeCodeVersions)
```

Comment:
It seems like some of the code review feedback I suggested last time is still missing. Not sure if that was an misunderstanding or a deliberate choice? Definitely there are options other than what I suggested but the logic still doesn't appear correct.

For the bindToAllNativeCodeVersions variable I suggested renaming it to offsetIsIL. Binding in all versions is already controlled by whether code->IsIL().

Reply:
This was an oversight, I'm correcting it now.